assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -175,9 +175,9 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
     .attr('height', availHeight)
     .attr('aria-hidden', 'true')
     .attr('transform', `translate(${margin.left}, ${margin.top})`)
-    .on('wheel.zoom', scroll)
+    .on('wheel.zoom', scroll, { passive: true })
```

Comment:
Usually we try to keep changes separate between issues this is touching Resource accessed. It is fine for now as this is doesn't have a huge effect and it is just UI improvement

Reply:
Yeah this was completely unrelated, I was just eliminating a warning message.  I will try to keep these to separate issues in the future.